<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsDialog</class>
 <widget class="QDialog" name="StatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="graphStackedWidget">
     <widget class="ScoreGraph" name="scorePage"/>
     <widget class="ResultsBarGraph" name="resultBarPage"/>
     <widget class="RelativeResultGraph" name="relativeResultPage"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="graphSettings">
       <property name="text">
        <string>Graph settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="prevGraphButton">
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="graphLabel">
       <property name="text">
        <string>Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextGraphButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScoreGraph</class>
   <extends>QWidget</extends>
   <header>ScoreGraph.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultsBarGraph</class>
   <extends>QWidget</extends>
   <header>ResultsBarGraph.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RelativeResultGraph</class>
   <extends>QWidget</extends>
   <header>RelativeResultGraph.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
